package.json

```diff
@@ -28,13 +28,14 @@
   "author": "",
   "license": "AGPL-3.0",
   "devDependencies": {
+    "@hermeznetwork/commonjs": "0.0.7",
```

Comment:
Please, rebase with master since `@hermeznetwork/commonjs` was upgraded to version 0.0.8: `https://github.com/hermeznetwork/contracts/blob/master/package.json#L62`